### PR TITLE
Porting v13 articles to release folder

### PR DESCRIPTION
**Pull request by jonat123**
## Description
This PR contains the changes for all product updated to v13, moved to a new folder which will be used for the release.
_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
13

## Deadline (if relevant)
14 of december
_When should the content be published?_
14 of december

### CODE REVIEW

File: 13-release/umbraco-forms/developer/prepping-frontend.md [hunk 1]
```diff
@@ -26,16 +26,6 @@ Alternatively, you can add the dependencies to the body tag:
...
-If you want to modify the rendering of the scripts, you can provide a object parameter named `htmlAttributes`. The contents of the object will be written out as HTML attributes on the script tags.
-
-You can use this to apply `async` or `defer` attributes. For example:
-
-```csharp
-@Html.RenderUmbracoFormDependencies(Url, new { @async = "async" })
-```
-
-If using `async`, please make sure to [disable the Forms client-side validation framework check](../developer/configuration/README.md#disableclientsidevalidationdependencycheck). This is necessary as it's not possible to guarantee that the asynchronous script will load in time to be recognized by the check. This can then cause a false positive warning.
-
```
Comment (alina-tincas) [inline 13-release/umbraco-forms/developer/prepping-frontend.md:38]: is this supposed to be removed? I can see it was introduced in v12: https://github.com/umbraco/UmbracoDocs/pull/5492/files maybe ask Andy?

File: 13-release/umbraco-forms/editor/creating-a-form/README.md [hunk 2]
```diff
@@ -74,26 +72,6 @@ When reordering your Form, you can drag and drop the fields to make it look the
...
-### Form Groups
-
-With a page, form fields can be arranged into groups. These will display all together on a single page but can be styled so the fields are appropriately grouped in fieldsets.
-
-New groups are added via the **Add new group** button.
-
-## Form Columns
-
-The last level of structure are columns that can be created within a group. To set the number of columns click the **cog** icon next to the group.  You will then be able to add or move fields to the new columns created.
-
```
[the remark this answers is not in the document]
Comment (jonat123) [reply 13-release/umbraco-forms/editor/creating-a-form/README.md:96]: No I dont think it is supposed to be removed, I will re-add it :)

File: 13-release/umbraco-forms/developer/prepping-frontend.md (hunk 1 above)
Comment (jonat123) [reply 13-release/umbraco-forms/developer/prepping-frontend.md:38]: No I dont think it is supposed to be removed, I will re-add it :)

File: 13-release/umbraco-cms/reference/content-delivery-api/extension-api-for-querying.md [hunk 3]
```diff
@@ -230,18 +230,6 @@ The principal difference from the selector is that the filter implements `BuildF
...
-To define your custom filter, you currently have four options: `Is`, `IsNot`, `Contains` and `DoesNotContain`.
-
-It is possible to pass multiple values to each operator, and these values will be treated inclusively as an __or__ operator. For example, if `tag1` and `tag2` were passed into a filter using the `Is` operator, _any_ document containing __either__ `tag1` __or__ `tag2` would return. The request for this might look like this:
-
-`GET /umbraco/delivery/api/v1/content?filter=customTagFilter:tag1,tag2`
-
-If you require this functionality to be restrictive i.e. `tag1` __and__ `tag2`, then the current approach would be to chain the custom filter. The request would change to look more like this:
-
-`GET /umbraco/delivery/api/v1/content?filter=customTagFilter:tag1&filter=customTagFilter:tag2`
-
```
Comment (alina-tincas) [inline 13-release/umbraco-cms/reference/content-delivery-api/extension-api-for-querying.md:244]: is this supposed to be removed? I can see on this PR: https://github.com/umbraco/UmbracoDocs/pull/5682/files#diff-f64d504745f1560b024cf3cf71341bb4d8668075d0c8d0d6964c8a7ccc77acab that I am gonna merge thurdsday that things are added on top of this 🤔

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Good job @jonat123 🙌 

I have noticed some things that are unclear, can you take a look at my review? 😊

File: 13-release/umbraco-commerce/key-concepts/ui-config-files.md [hunk 4]
```diff
@@ -117,7 +117,6 @@ The Order Line config block configures which Order Line properties should be vie
 | `alias`                  | The alias of the Order Line property                                                                                               |
 | `label`                  | A friendly label to display for this property in the editor interface                                                              |
 | `description`            | A friendly description to display for this property in the editor interface                                                        |
-| `template`               | An angular template to use for rendering the property value. Defaults to `{​{ properties['alias'].value }​}`                         |
```
[the remark this answers is not in the document]
Comment (jonat123) [reply 13-release/umbraco-commerce/key-concepts/ui-config-files.md:120]: Good question, I will add it back!

File: 13-release/umbraco-commerce/key-concepts/ui-config-files.md [hunk 5]
```diff
@@ -285,7 +284,6 @@ The Additional Info config block configures any other Cart/Order properties you
 | `alias`       | The alias of the Order property                                                                                               |
 | `label`       | A friendly label to display for this property in the editor interface                                                         |
 | `description` | A friendly description to display for this property in the editor interface                                                   |
-| `template`    | An angular template to use for rendering the property value. Defaults to `{​{ properties['alias'].value }​}`                    |
```
[the remark this answers is not in the document]
Comment (jonat123) [reply 13-release/umbraco-commerce/key-concepts/ui-config-files.md:288]: Same as above, I have just added it back :)

File: 13-release/umbraco-cms/reference/content-delivery-api/extension-api-for-querying.md (hunk 3 above)
Comment (jonat123) [reply 13-release/umbraco-cms/reference/content-delivery-api/extension-api-for-querying.md]: Good catch!

I have added it back :D

File: 13-release/umbraco-deploy/handling-cache-refresher-notifications.md [hunk 6]
```diff
@@ -1,168 +0,0 @@
----
-meta.Title: "Handling Cache Refresher Notifications"
-description: "How to respond to deployment events using cache refresher notifications"
----
-
-# Handling Cache Refresher Notifications
-
```
[the remark this answers is not in the document]
Comment (jonat123) [reply 13-release/umbraco-deploy/handling-cache-refresher-notifications.md]: It should indeed! Adding it back.

**Pull request merged**